src/Type/Definition/ImplementingType.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GraphQL\Type\Definition;
+
+/*
+export type GraphQLImplementingType =
+GraphQLObjectType |
+GraphQLInterfaceType;
+*/
+
+interface ImplementingType
+{
+    public function implementsInterface(InterfaceType $interfaceType) : bool;
+
+    /**
+     * @return InterfaceType[]
```

Comment:
```suggestion
     * @return array<int, InterfaceType>
```

...assuming the result is not supposed to be a map?

Reply:
This has the same return type as `ObjectType:getInterfaces` (and `ObjectType::$interfaces`) since that's what this interface is based on (and then implemented by both `ObjectType` and `InterfaceType`).

Reply:
Alright, then my suggestion seems right.

Reply:
Cool, I'll include the change for the ObjectType as well then, since it implements the interface.